### DISCUSSION TITLE
Skull & Bones

### DISCUSSION THREAD

**Discussion by Mrtritolo** [gameserverapp/Platform#765] [Game request]
I couldn't find any other requests so I'll create it, it would be nice to have Skull & bones on GSA

**Comment by LegitGammin** [comment]
Skull and bones have a dedicated server feature?

**Reply by Mrtritolo** [reply]
i'm not sure but other provider will sell this service in future https://shorturl.at/gkQY6

**Comment by gnarlyvr** [comment]
Skull and Bones doesn't support dedicated servers and haven't announced or hinted at that happening.
